docs/coligo.md

```diff
@@ -0,0 +1,101 @@
+# PyWren on IBM Coligo (Experimental)
+
+**Notice that IBM Coligo is still in beta. Use this only for testing**
+
+IBM Coligo is based on Knative, so the only requirement to make it working is to have the KUBECONFIG file properly configured. To do so, follow the next steps:
+
+### Installation
+
+1. Install the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli):
+
+   ```bash
+   curl -sL https://ibm.biz/idt-installer | bash
+   ```
+
+2. Login to your account (IBM Coligo is currently present on us_south region, so login to this region)
+
+   ```bash
+   ibmcloud login -r us_south
+   ```
+
+3. Install the IBM Coligo plugin:
+
+   ```bash
+   ibmcloud plugin install coligo
+   ```
+
+4. Create a new Coligo project (you can also do this through the dashboard):
+
+   ```bash
+   ibmcloud coligo project create --name myproject
+   ```
+
+5. Target to this project:
+
+   ```bash
+   ibmcloud coligo target --name myproject
+   ```
+   After running the target command, you will see in your screen the location of the KUBECONFIG file of your Coligo project. Set the KUBECONFIG environment variable, for example:
+
+   ```bash
+   export KUBECONFIG=/home/myuser/.bluemix/plugins/coligo/myproject-b59a1c9f-5ds6-j1sm5.yaml
+   ```
+
+7. [Install the Docker CE version](https://docs.docker.com/get-docker/) (PyWren needs to built the default runtime the first time you run a script)
+
+8. Login to your docker account:
+   ```bash
+   docker login
+   ```
+
```

Comment:
@JosepSampe can you please put here new section naming 
"Access Coligo via Knative API" and then all knative stuff + example will be in this section.
I will then add new section how to use Job Descriptor API of Coligo and document my stuff in pywren-betabs branch

Reply:
In fact all the document is for knative API. Do you have common steps? If no, maybe you can simply add your instructions at the end of the document

Reply:
@JosepSampe yeah..there are common steps. I will merge it and then modify a bit. thanks